README.md

```diff
@@ -8,102 +8,197 @@
+2. You can check you did it right by running `latex` 
+
+#### Additional Installation Steps
+- You must install additional codecs to play MP4 files:
+```
+sudo apt install libdvdnav4 libdvd-pkg gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly libdvd-pkg
+sudo apt install ubuntu-restricted-extras
 ```
 
-For more options, take a look at the [Using manim](#using-manim) sections further below.
+### Mac
 
-### Directly
+Before installing `manim-community`, there are some additional dependencies that you must have installed:
+ - Cairo
+ - FFmpeg
+ - Sox (optional, for sound)
+ - LaTeX (optional, for LaTeX)
+ - Homebrew
 
-If you want to hack on manimlib itself, clone this repository and in that directory execute:
+#### Brew Installation
 
-```sh
-# Install python requirements
-python3 -m pip install -r requirements.txt
+HomeBrew is a package manager similar to `apt` on linux that allows you to install packages easily.
+If it is not installed, please install it with this command:
 
-# Try it out
-python3 ./manim.py example_scenes.py SquareToCircle -pl
+```
+ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" < /dev/null 2> /dev/null
 ```
 
-### Directly (Windows)
-1. [Install FFmpeg](https://www.wikihow.com/Install-FFmpeg-on-Windows).
-2. [Install Cairo](https://www.lfd.uci.edu/~gohlke/pythonlibs/#pycairo). For most users, ``pycairo‑1.18.0‑cp37‑cp37m‑win32.whl`` will do fine.
-    ```sh
-    pip3 install C:\path\to\wheel\pycairo‑1.18.0‑cp37‑cp37m‑win32.whl
-    ```
-3. Install a LaTeX distribution. [MiKTeX](https://miktex.org/download) is recommended.
+#### Cairo Installation
+1. Run `brew install cairo`
 
-4. [Install SoX](https://sourceforge.net/projects/sox/files/sox/).
+#### FFmpeg Installation
+1. Refer to the section on Brew installation to see how to install `brew`
+2. Run `brew install ffmpeg`
+3. You can check you did it right by running `ffmpeg -version`
 
-5. Install the remaining Python packages. Make sure that ``pycairo==1.17.1`` is changed to ``pycairo==1.18.0`` in requirements.txt.
-    ```sh
-    git clone https://github.com/3b1b/manim.git
-    cd manim
-    pip3 install -r requirements.txt
-    python3 manim.py example_scenes.py SquareToCircle -pl
-    ```
+#### SoX Installation
+1. Refer to the section on Brew installation to see how to install `brew`
+2. Run `brew install sox`
+3. You can check you did it right by running `sox`
 
+#### LaTeX Installation
+1. Refer to the section on Brew installation to see how to install `brew`
+2. Run `brew cask install mactex`
+2. You can check you did it right by running `latex` 
 
-## Anaconda Install
+### Installing Manim-Community itself
 
-* Install sox and latex as above. 
-* Create a conda environment using `conda env create -f environment.yml`
-* **WINDOWS ONLY** Install `pyreadline` via `pip install pyreadline`. 
+Manim runs on Python 3.7+. If you'd like to just use the library, you can install it from PyPI via pip:
 
+```sh
+pip3 install manimlib
```

Comment:
Does this work without a setup.py?

Reply:
I don't think so

Reply:
Then this should be removed first.

Reply:
I believe someone in the discord said to keep this here, should I just remove it?

Reply:
It's fine then, we should have a setup.py shortly